src/api/VoteMonitor.Api.Note/Controllers/NoteController.cs

```diff
@@ -29,12 +28,15 @@ public NoteController(IMediator mediator, IMapper mapper)
 
 
         [HttpGet]
-        public async Task<List<NoteModel>> Get(NoteQuery filter)
+        public async Task<IActionResult> Get(NoteQuery filter)
         {
+            if (filter.IdQuestion.HasValue && !filter.IdPollingStation.HasValue)
+                return BadRequest($"If the {filter.IdQuestion} param is provided then the {filter.IdPollingStation} param is required !");
```

Comment:
`return BadRequest("If the IdQuestion param is provided then the IdPollingStation param is required !");` ?

Reply:
Oups, forgot to add nameof(), fixed.